src/test/java/LoginTests.java

```diff
@@ -0,0 +1,54 @@
+
+import Pages.LoginPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class LoginTests extends BaseTests {
+    @Test
+    void loginSuccessTest() {
+        //1. Login Success.
+
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.openLoginPage();
+        loginPage.enterEmail("alekseimaslo@gmail.com");
+        loginPage.enterPassword("Kwidos123");
+        loginPage.clickSubmit();
+        WebElement verifiedUserName = driver.findElement(By.xpath("//*[contains (text(), 'Aleksei')]"));
```

Comment:
If you use page object you should create another Page class which appears after logged in and hide this locator and findElement method there